data/src/test/java/com/microsoft/azure/kusto/data/ClientRequestPropertiesTest.java

```diff
@@ -0,0 +1,34 @@
+package com.microsoft.azure.kusto.data;
+
+import org.json.JSONException;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.skyscreamer.jsonassert.JSONAssert;
+
+import java.util.concurrent.TimeUnit;
+
+public class ClientRequestPropertiesTest {
+    @Test
+    @DisplayName("test set/get timeout")
+    void testSetGetTimeout() {
```

Comment:
I'd avoid using the test prefix - everything in this class are tests, and also have the "@test" notation...
maybe name it timeout_SetGet